src/xr/context_object.py

```diff
@@ -266,6 +263,10 @@ def view_loop(self, frame_state):
                     space=self.space,
                 )
             )
+            num_views = len(views)
+            projection_layer_views = (xr.CompositionLayerProjectionView * num_views)(
+                *([xr.CompositionLayerProjectionView()] * num_views))
```

Comment:
I'm concerned that `[xr.CompositionLayerProjectionView()] * num_views)` might create an array of multiple references to a single CompositionLayerProjectionView object. Perhaps a list comprehension like `[xr.CompositionLayerProjectionView() for _ in range(num_views)]` would avoid that problem.

Reply:
Thanks, fixed now.